twitter_ads/creative.py

```diff
@@ -190,8 +200,6 @@ class VideoWebsiteCard(Resource, Persistence):
 resource_property(VideoWebsiteCard, 'video_poster_width', readonly=True)
 resource_property(VideoWebsiteCard, 'video_url', readonly=True)
 resource_property(VideoWebsiteCard, 'video_width', readonly=True)
-resource_property(VideoWebsiteCard, 'website_dest_url', readonly=True)
```

Comment:
It looks like `website_dest_url` and `website_display_url` *are* returned in the response. The documentation just doesn't reflect that yet. Let's add these back in